src/main/definitions/case.ts

```diff
@@ -12,7 +12,12 @@ export interface CaseDate {
 }
 
 export interface Case {
-  dobDate: CaseDate
+  dobDate: CaseDate;
+  address1?: string;
```

Comment:
Can the whole address be null? Or do we need at least one non-nullable address field?

Reply:
This was more part of my PR. For the `Case` to work we only need to have one field as mandatory and everything else can be optional.